src/main/java/com/wavefront/helpers/Defaults.java

```diff
@@ -14,22 +15,25 @@
  */
 public class Defaults {
   private static final Random RANDOM = new Random(System.currentTimeMillis());
+  public static final String SERVICE = "service";
+  public static final String ERROR = "error";
+  public static final String DEBUG = "debug";
+  public static final String DEFAULT_TYPE_NAME_PREFIX = "traceType_";
+  public static final String DEFAULT_SPAN_NAME_SUFFIX = "abcdefghijklmnopqrstuvxyz";
+  public static final int DEFAULT_NESTING_LEVEL = RANDOM.nextInt(6) + 4;
+  public static final Pair<String, String> ERROR_TAG = new Pair<>(ERROR, "true");
+  public static final Pair<String, String> DEBUG_TAG = new Pair<>(DEBUG, "true");
 
-  public static final List<TraceTypePattern.TagVariation> DEFAULT_MANDATORY_TAGS = List.of(
-      new TagVariation("application", List.of("Application_1")),
-      new TagVariation("cluster", List.of("us-west")),
-      new TagVariation("shard", List.of("primary")),
-      new TagVariation("service", List.of("Service_1", "Service_2")));
+  public static final List<TagVariation> DEFAULT_MANDATORY_TAGS = List.of(
+      new TagVariation("application", Set.of("Application_1")),
+      new TagVariation("cluster", Set.of("us-west")),
+      new TagVariation("shard", Set.of("primary")),
```

Comment:
Could you please define variables for string values  "application", "cluster", "shard"